package.json

```diff
@@ -34,59 +32,53 @@
     }
   },
   "lint-staged": {
-    "*.js": [
-      "eslint --fix",
-      "flow focus-check"
+    "*.ts": [
+      "eslint --fix"
     ],
-    "*.{js,json,md}": [
+    "*.{json,md}": [
```

Comment:
I think we want to run prettier on our ts files too

Reply:
yup, will add .ts